project_4/index.html

```diff
@@ -0,0 +1,384 @@
+                    <p></p>
+                </div>
+
+                <p class="schedule-time talk-time">3:00pm</p>
+                <div class="schedule-info talk-header">
+                    <p>3:00-4:30pm</p>
+                    <p class="schedule-title">Talk: Ecotech Myths and Lessons from the Past</p>
+                    <p>The present-day approach towards a sustainable society is doomed to fail. The focus on
+                        sophisticated technology – electric and hybrid cars, energy-efficient devices, solar panels and
+                        wind turbines, for instance – has little or no effect because these green technologies require
+                        large amounts of energy and resources for their manufacture, which makes their development
+                        highly dependent on a continuous supply of fossil fuels. What we need to solve our problems is
+                        exactly the opposite: less sophisticated technology. There is a lot to learn from the past.
+                        While they often worked surprisingly good, most of low-tech solutions have been completely
+                        forgotten. In his speech, Kris de Decker explains that reverting to past technologies does not
+                        mean that we should go back to the Middle Ages. Rather, it means combining old tech with new
+                        knowledge and new materials, or applying old concepts and lost knowledge to modern technology.
+                        <!-- source for talk information: http://transitionbrockville.com/2016/05/16/ecotech-myths-and-lessons-from-the-past/ -->
+                    </p>
+                </div>
+
+                <p class="schedule-time">4:30pm</p>
+                <div class="schedule-info">
+                    <p>4:30-5:30pm</p>
+                    <p class="schedule-title">Group Discussion</p>
+                    <p>We will break out into our designated small groups and hold
+                        discussions about the talk led by our discussion leaders.</p>
+                </div>
+
+                <p class="schedule-time">5:30pm</p>
+                <div class="schedule-info">
+                    <p>5:30-6:00pm</p>
+                    <p class="schedule-title">30 Minute Break</p>
+                </div>
+
+                <p class="schedule-time">6:00pm</p>
+                <div class="schedule-info">
+                    <p>6:00-9:00pm</p>
+                    <p class="schedule-title">Closing Party</p>
+                    <p>Enjoy dinner, drinks, and dancing in the main hall for the closing confernce party</p>
+                </div>
+            </div>
+        </section>
+
+    </main>
+
+    <!-- REGISTER -->
+    <div class="register">
+        <p class="register-text">Register now to network, learn, and become a part of the sustainability narrative.
+            Spots are limited. We can't wait to see you there!</p>
+        <a href="">Purchase your ticket</a>
```

Comment:
Browsers can be weird about blank `href`s -- I'd include something here, even if it shouldn't go anywhere:
```suggestion
        <a href="#">Purchase your ticket</a>
```